### ISSUE TITLE
form error messages are not accessible (not read by screen reader)

### ISSUE THREAD

**Issue by asafcode** [vuetifyjs/vuetify#13379]
The error messages associated with form inputs are not read by screen readers.

to fix this: label must wrap both the input and the error message 
OR 
assign an aria-described-by attribute to the input, set its value to the id of the error message element.

https://vuejsdevelopers.com/2019/05/13/accessibility-validation-aria-vuejs/
